resources/views/upload-product.blade.php

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        <form action="{{route('products.store')}}", method="POST" enctype="multipart/form-data">
+            @csrf
+            <label for="prod_name">Product name:</label>
+            <input type="text" id="prod_name" name="name" required>
+
+            <label for="prod_description">Description:</label>
+            <input type="text" name="description" id="prod_description" required>
+
+            <label for="prod_price">Price:</label>
+            <input type="number" name="price" id="prod_price" step="any" required>
+
+            <label for="prod_category">Category</label>
+            <input type="text" name="category" id="prod_category" required>
```

Comment:
Considering the changes above, this will become a select form input. Then loop through one option tag and add the value and option for each instance in the array of categories